installer/charts/tssc-dh/templates/backstage.yaml

```diff
@@ -23,6 +23,13 @@ spec:
       spec:
         template:
           spec:
+            containers:
+              - name: backstage-backend
+                resources:
+                  requests:
+                    cpu: 250m
```

Comment:
Not a fan of hardcoding resources params, but rather provide defaults and allow user to determine their own size, can this be moved to config file or at least values file? Can you think about the other way, how do we know that this is not too restrictive? Would 4 CPUs be a better limit? What about the memory?

Reply:
Also here: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.7/html/configuring_red_hat_developer_hub/configuring-the-deployment

So the request seems reasonable